packages/validation/src/types.ts

```diff
@@ -0,0 +1,25 @@
+export type Condition = Value | RichCondition;
+
+export interface ExtSchemaContext extends SchemaContext {
+  comparableField?: string;
+  comparableValue?: Value;
+  isString?: boolean;
+  maxVal?: number;
+  minVal?: number;
+  schemaValue?: Value;
+}
+
+export interface RichCondition {
+  msg: string;
+  value: Value;
+}
+
+export interface SchemaContext {
+  fieldName: string;
+  schema: any;
+  values: any;
+}
+
+export type Value = string | number | boolean;
+
+export type Validator = string | undefined;
```

Comment:
This export use none of the other files. Do we really need it?

Reply:
You are right.